### ISSUE TITLE
Question about how to prepare the input of numbers

### ISSUE THREAD

**Issue by JonasLi-19** [lucidrains/linformer#9]
Hi!  my project is to predict a number according to a long sequence of numbers(a csv with 10000 columns, most of them are 0, some of them are int numbers) So it is a simple regression task. Is it appropriate to use Linformer here?

**Comment by JonasLi-19** [comment]
My classmate told me that I should consider each sample as a word, so that 10000 features could be the dim of the sample, because one sample only refer to one number result. And I need to input multiple samples at once, he calls it 'each time step'.
What do you think of his suggestion? I would be glad to get your suggestions~~

**Comment by JonasLi-19** [comment]
Hi dear developer, I wanna share more specific info about my question for you to discuss~
Each biology sample in my input csv file contains 10,000 (125 features * 80 distance) features columns, and one float number as the label. My friend suggested that **every 125 features can be considered as a block representing a 'word'**, so I need to find a way to represent 125 features into a embedding vector? And each feature is an integer (most of which are zeros). Then, each sample is a 'sentence' with `seq_lens = 80 `which corresponds to a label.

In summary, I am wondering if this task suitable for Linformer, and how to set the 125 features into a 'word'.

Sorry to bother you but our biotech group doesn't have anyone who is majored in AI.Thanks!
